app/code/Magento/Sales/Ui/Component/Listing/Column/Price.php

```diff
@@ -51,13 +53,9 @@ public function prepareDataSource(array $dataSource)
         if (isset($dataSource['data']['items'])) {
             foreach ($dataSource['data']['items'] as & $item) {
                 $currencyCode = isset($item['base_currency_code']) ? $item['base_currency_code'] : null;
-                $item[$this->getData('name')] = $this->priceFormatter->format(
-                    $item[$this->getData('name')],
-                    false,
-                    null,
-                    null,
-                    $currencyCode
-                );
+                $basePurchaseCurrency = $this->currencyFactory->create()->load($currencyCode);
```

Comment:
Since the `load()` method is deprecated, what do you think about using the following code to initialize the `$basePurchaseCurrency` variable?

```
$basePurchaseCurrency = $this->currencyFactory->->create(['data' => ['currency_code' => $currencyCode]]);
```

Reply:
![image](https://user-images.githubusercontent.com/8234209/62197175-96103400-b3a9-11e9-866f-f6e85c0e5cfc.png)

It is not deprecated. Please check it.

Reply:
Sorry, my fault, I didn't explain it well.

It's deprecated in the `\Magento\Framework\Model\AbstractModel` parent and my IDE also indicated it as deprecated.

> @deprecated 100.1.0 because entities must not be responsible for their own loading.             
Service contracts should persist entities. Use resource model "load" or collections to implement service contract model loading operations.                                                      

Actually, the `\Magento\Directory\Model\Currency`'s `load()` method don't really load from a Resource Model.

It was just a suggestion, for me it's ok to keep going like you did.